docs/src/modules/components/ApiPage/table/SlotsTable.tsx

```diff
@@ -88,11 +88,11 @@ export default function SlotsTable(props: SlotsTableProps) {
           </tr>
         </thead>
         <tbody>
-          {slots.map((params) => {
+          {slots.map((params, index) => {
             const { description, className, name, defaultValue, hash } = params;
 
             return (
-              <tr key={className} id={hash}>
+              <tr key={`${className}-${index}`} id={hash}>
```

Comment:
Some components don't have specific className for each slot ([like textfield](https://mui.com/material-ui/api/text-field/#slots)), so just using `className` as `key` is creating noise in docs if multiple slots doesn't have `className`

Reply:
Maybe the `hash` would be a better option since having two components with the same id would also be an issue